framework/src/base/MooseApp.C

```diff
@@ -278,6 +281,24 @@ MooseApp::~MooseApp()
 #endif
 }
 
+std::string
+MooseApp::getFrameworkVersion() const
+{
+  return MOOSE_VERSION;
+}
+
+std::string
+MooseApp::getVersion() const
+{
+  return MOOSE_VERSION;
+}
+
+std::string
+MooseApp::getPrintableVersion() const
+{
+  return type() + " Version: " + getVersion();
```

Comment:
Since you changed `MooseApp::name()` to be const, I am assuming you wanted to call `name()` instead of `type()` here?

Reply:
name() gives you the block name, type() gives you the app name

Reply:
The output of this is now: `MooseTestApp Version: 2b02e6c` on the test box. So, for example in RELAP-7 it would give me `RELAP7App Version: <sha1>`. I thought we wanted app name, not the name of this class.